src/models2/PostModel.es6.js

```diff
@@ -33,7 +33,11 @@ export default class PostModel extends RedditModel {
     name: T.string,
     over18: T.bool,
     postHint: T.string,
+    original_link: T.string,
```

Comment:
I think it would be more consistent if these were camel cased. If you make the switch I'd recommened moving them to the aliases section on line 52 and adding them to the `API_ALIASES` object